charts/apisix/README.md

```diff
@@ -69,6 +69,9 @@ The following tables lists the configurable parameters of the apisix chart and t
 | `apisix.podAnnotations`                  | Annotations to add to each pod                      | `{}`                                                    |
 | `apisix.podSecurityContext`              | Set the securityContext for Apache APISIX pods      | `{}`                                                    |
 | `apisix.securityContext`                 | Set the securityContext for Apache APISIX container | `{}`                                                    |
+| `apisix.podDisruptionBudget.enabled`     | Enable or disable podDisruptionBudget               | `false`                                                 |
+| `apisix.podDisruptionBudget.minAvailable`| Set the minAvailable of podDisruptionBudget         | `Not set`                                               |
```

Comment:
Should tell readers that `minAvailable` and `maxUnavailable` should be set exclusively, if both of them were set, `minAvailable` takes the precedence.

Also, let's add a hyperlink for Pod Disruption Budget.

Reply:
DONE